src/main/app.js

```diff
@@ -84,10 +84,20 @@ if (!lock) {
   app.quit();
 }
 
-app.on('second-instance', () => {
-  const focusedWindow = getMainWindow();
-  if (!focusedWindow) {
-    windowManager.buildWindowOfType('main');
+app.on('second-instance', (event, args) => {
+  reopenMainWindow(() => {
```

Comment:
Noticed that you don't check the `getMainWindow()` result now - what does `reopenMainWindow` do if there's already another app running? I checked the code briefly not not sure as to how to read it exactly. Does it simply ignore the other instance and open anyway?

Reply:
@perry-mitchell yeah this second instance thing is a bit tricky to read in general. So how this works is, if one instance is open, second instance simply doesn't open at all (it quits) and this `second-instance` event is fired in the *first* instance. So when this even is triggered, we simply bring back the main window if it's out of focus

Reply:
@perry-mitchell yeah this second instance thing is a bit tricky to read in general. So how this works is, if one instance is open, second instance simply doesn't open at all (it quits) and this `second-instance` event is fired in the *first* instance. So when this even is triggered, we simply bring back the main window if it's out of focus